tests/unit-tests/awsconfigs/common/aws-secrets-manager/test_data/expected/secrets-store.csi.x-k8s.io_v1alpha1_secretproviderclass_s3-secret.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
+kind: SecretProviderClass
+metadata:
+  name: s3-secret
+  namespace: kubeflow
+spec:
+  parameters:
+    objects: "- objectName: \"s3-secret\"\n  objectType: \"secretsmanager\"\n  jmesPath:\n
```

Comment:
what is this static creds doing here ?

Reply:
these are missing unit test updates from some of the previous PRs. the unit tests wont pass because one side of assertion was missing (actual output == **missing expected value**).

This could have been separate PR